spyder/widgets/fileswitcher.py

```diff
@@ -19,6 +20,8 @@
 # Local imports
 from spyder.config.base import _
 from spyder.py3compat import iteritems, to_text_string
+from spyder.config.fonts import FILESWITCHER_FONT
```

Comment:
@juanis2112 please remove this line is not needed anymore and it is the cause of the failing tests :)